lib/questionnaires/model/item/answer/src/coding_answer_option_model.dart

```diff
@@ -154,6 +154,7 @@ class CodingAnswerOptionModel {
         forDisplay = _createForDisplay(coding, locale, questionnaireItemModel);
         optionText = RenderingString.fromText(
           plainText,
+          extensions: extensions,
```

Comment:
I'm also not sure about this one either. Here `plainText` is not coming directly from the question text itself, but from whatever is being returned by `_createMultiColumn` in line 153, so we may end up replacing the actual content when the text is translated.

Reply:
It's been addressed [here](https://github.com/sujrd/faiadashu/pull/49/commits/3598028e169d969b943d32e8b739ed21aebcca11), thanks!